pipelines/folioRancher/folioScheduledTesting/karateTestsScheduledPipeline.groovy

```diff
@@ -61,9 +61,14 @@ pipeline {
         stage("Create environment") {
             steps {
                 script {
-                    def jobParameters = getEnvironmentJobParameters('apply', okapiVersion, clusterName,
-                        projectName, prototypeTenant, folio_repository, folio_branch)
-                    spinUpEnvironmentJob = build job: spinUpEnvironmentJobName, parameters: jobParameters, wait: true, propagate: false
+                    try {
+                        def jobParameters = getEnvironmentJobParameters('apply', okapiVersion, clusterName,
+                            projectName, prototypeTenant, folio_repository, folio_branch)
+                        spinUpEnvironmentJob = build job: spinUpEnvironmentJobName, parameters: jobParameters, wait: true, propagate: false
+                    } catch (Exception new_ex) {
+                        println('Failed to create environment: ' + new_ex)
+                        slackNotifications.sendPipelineFailSlackNotification("#rancher_tests_notifications")
```

Comment:
Here you need to propagate the error as after slack notification, the send pipeline will continue execution of the following stages.

Reply:
I've added the "throw new Exception"